Sources/StreamChat/Workers/MessageUpdater.swift

```diff
@@ -825,10 +834,10 @@ extension MessageUpdater {
         }
     }
     
-    func editMessage(messageId: MessageId, text: String, skipEnrichUrl: Bool, attachments: [AnyAttachmentPayload] = [], extraData: [String: RawJSON]? = nil) async throws {
+    func editMessage(messageId: MessageId, text: String, skipEnrichUrl: Bool, attachments: [AnyAttachmentPayload] = [], extraData: [String: RawJSON]? = nil) async throws -> ChatMessage {
```

Comment:
general remark: formatting can be improved in few places, the parameters should go into new lines.

Reply:
Going to be a separate change, I need reformat many files